src/bin/pg_autoctl/supervisor.c

```diff
@@ -656,7 +1134,7 @@ supervisor_restart_service(Supervisor *supervisor, Service *service, int status)
 	 */
 	if (service->policy == RP_TEMPORARY)
 	{
-		return true;
+		return false;
```

Comment:
Should that be part of this PR? It looks like a bug fix that is worthwhile anyway?

Reply:
It's being a while since I coded that bit. It may well have been a bug fix that snuck in. Let me revisit the code and come back :)